README.md

```diff
@@ -1,4 +1,4 @@
-CongoMongo [![ClojarsVersionRelease](https://img.shields.io/clojars/v/congomongo)](https://clojars.org/congomongo) [![ClojarsDownloads](https://img.shields.io/clojars/dt/congomongo)](https://clojars.org/congomongo) [![CongoMongo](https://circleci.com/gh/congomongo/congomongo.svg?style=svg)](https://circleci.com/gh/congomongo/congomongo)
+#CongoMongo [![ClojarsVersionRelease](https://img.shields.io/clojars/v/congomongo)](https://clojars.org/congomongo) [![ClojarsDownloads](https://img.shields.io/clojars/dt/congomongo)](https://clojars.org/congomongo) [![CongoMongo](https://circleci.com/gh/congomongo/congomongo.svg?style=svg)](https://circleci.com/gh/congomongo/congomongo)
```

Comment:
The ======= immediately below already denotes this as a level-1 heading, so this is not necessary.